src/Tribe/Rewrite.php

```diff
@@ -559,12 +565,13 @@ protected function get_handled_rewrite_rules() {
 			$this->setup();
 		}
 
+		$all_rules     = isset( $this->rewrite->rules ) ? (array) $this->rewrite->rules : [];
+
 		if ( null === $our_rules ) {
 			// While this is specific to The Events Calendar we're handling a small enough post type base to keep it here.
 			$pattern = '/post_type=tribe_(events|venue|organizer)/';
 			// Reverse the rules to try and match the most complex first.
-			$rules     = isset( $this->rewrite->rules ) ? (array) $this->rewrite->rules : [];
-			$our_rules = array_filter( $rules,
+			$our_rules = array_filter( $all_rules,
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 696be60989b7fad6afd991aaf728f48b</sub>